bower.json

```diff
@@ -1,6 +1,5 @@
 {
   "name": "jquery.cookie",
-  "version": "1.4.1",
```

Comment:
This is moot, see https://github.com/bower/spec/commit/a325da3 and https://github.com/carhartl/jquery-cookie/pull/365